src/Orleans/GrainDirectory/MultiClusterStatus.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.GrainDirectory
+{
+    /// <summary>
+    /// Status of a directory entry with respect to multi-cluster registration
+    /// </summary>
+    internal enum MultiClusterStatus
```

Comment:
The idea currently is to use "Owned" in that case, which gives the right behavior. We could consider using "None" as well.